octobot_trading/personal_data/trades/trades_manager.py

```diff
@@ -53,6 +53,12 @@ def upsert_trade_instance(self, trade):
             self.trades[trade.trade_id] = trade
             self._check_trades_size()
 
+    def has_closing_trade_with_order_id(self, order_id) -> bool:
+        for trade in self.trades.values():
```

Comment:
I would have preferred that we avoided that. It can be a "big" computation if we have a lot of trades in history. I don't know how to improve that.